weed/server/filer_grpc_server_sub_meta.go

```diff
@@ -62,8 +63,19 @@ func (fs *FilerServer) SubscribeMetadata(req *filer_pb.SubscribeMetadataRequest,
 			return nil
 		}
 
+		glog.V(4).Infof("processed to %v: %v", clientName, processedTsNs)
 		if processedTsNs != 0 {
 			lastReadTime = log_buffer.NewMessagePosition(processedTsNs, -2)
+		} else {
+			nextDayTs := util.GetNextDayTsNano(lastReadTime.UnixNano())
+			position := log_buffer.NewMessagePosition(nextDayTs, -2)
+			found, err := fs.filer.HasPersistedLogFiles(position)
```

Comment:
this only check whether there are logs in the next day.

What if there are multiple days without any logs?

Reply:
> this only check whether there are logs in the next day.
> 
> What if there are multiple days without any logs?

Right, we also need to consider this.

Reply:
If there are multiple days without any logs, the for loop should loop every day, until today? I can verify this in our teststand.

Reply:
Tested in our env that, last weekend there is no IO for 2 days, today we write some files and new events can still be subscribed with no problems.